drivers/virtio/Kconfig

```diff
@@ -0,0 +1,33 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig DRIVERS_VIRTIO
+	bool "Virtio Device Support"
+	default n
+
+menuconfig DRIVERS_VIRTIO_BASE
+	hex "Virtio-mmio base address"
+	default 0x0
```

Comment:
how to support multiple mmio device instance?

Reply:
@xiaoxiang781216 
I think multiple mmio devices support is future work.

Reply:
why not let caller pass the address to virtio_init to resolve this issue? I think it's better to address the architecture issue in early stage as much as possible.

Reply:
@masayuki2009 add multiple mmio device instance support is important for the virtio architecture and maybe we need to consider it early. otherwise we will rework the driver if the architecture is change

Reply:
@xiaoxiang781216 @qinwei2004 
OK, let me consider supporting multiple device instances.